### PR TITLE
Added not about git subtree merging

### PR DESCRIPTION
**Pull request by RosaWagner**
@vv-monsalve @chrissimpkins @m4rc1e Is this the correct wording and all ?

### CODE REVIEW

File: gf-guide/googlefonts.md [hunk 1]
```diff
@@ -115,6 +115,9 @@ This directory is a git subtree that is automatically updated with changes in th
 git subtree pull --prefix=axisregistry https://github.com/googlefonts/axisregistry.git main
 ```
 
+**Very important note:** the PR landing the gitsubtree into google/fonts repo should be merged in the main branch using **merge commits** (and _not_ squash and merge).
```
Comment (m4rc1e) [inline gf-guide/googlefonts.md:118]:
```suggestion
**Important:** When merging a pull request in the google/fonts repo which contains a subtree pull, merge it using the "Create a merge commit" button and not the "Squash and merge" button. If we squash and merge, the subtree's history is squashed into a single commit which will cause merge conflicts for the next person who has to do a subtree pull. 
```

File: gf-guide/googlefonts.md [hunk 2]
```diff
@@ -130,6 +133,8 @@ This directory is a git subtree that is automatically updated with changes in th
 git subtree pull --prefix=lang https://github.com/googlefonts/lang.git main
 ```
 
+**Very important note:** the PR landing the gitsubtree into google/fonts repo should be merged in the main branch using **merge commits** (and _not_ squash and merge).
```
Comment (m4rc1e) [inline gf-guide/googlefonts.md:136]:
```suggestion
**Important:** When merging a pull request in the google/fonts repo which contains a subtree pull, merge it using the "Create a merge commit" button and not the "Squash and merge" button. If we squash and merge, the subtree's history is squashed into a single commit which will cause merge conflicts for the next person who has to do a subtree pull. 
```

**Comment by m4rc1e** [comment]
LGTM.

I've changed the wording a bit and described what will happen if they don't do it. Your call to accept what I've done. If you prefer yours, please feel free to merge.

**Comment by RosaWagner** [comment]
Super, thanks!

**Comment by chrissimpkins** [comment]
Looks great! Thanks Rosalie

**Pull request merged**